app/code/Magento/Inventory/Model/GetReservationQuantityInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model;
+
+/**
+ * Responsible for retrieving Reservation Quantity (without stock data)
+ *
+ * @api
+ */
+interface GetReservationsQuantityInterface
```

Comment:
we no need this interface, as we no need to introduce extension point at this level, it's very confusing for developers.